src/Classes/Loader.ts

```diff
@@ -50,59 +48,96 @@ export class Loader {
         return Loader.load(tagId, params[idx]);
     }
 
+    /**
+     * Loads an options object from the provided array to create a [[Container]] object.
+     * @param id the container id
+     * @param domContainer the dom container for keeping
+     * @param params the options array to get the item from
+     * @param index if provided gets the corresponding item from the array
+     */
+    public static async setFromArray(id: string,
+                                     domContainer: HTMLElement,
+                                     params: RecursivePartial<IOptions>[],
+                                     index?: number): Promise<Container | undefined> {
+        let idx: number;
+
+        if (index === undefined || index < 0 || index >= params.length) {
+            idx = Math.floor(Math.random() * params.length);
+        } else {
+            idx = index;
+        }
+
+        return Loader.set(id, domContainer, params[idx]);
+    }
+
     /**
      * Loads the provided options to create a [[Container]] object.
      * @param tagId the particles container element id
      * @param params the options object to initialize the [[Container]]
      */
-    public static load(tagId: string, params?: RecursivePartial<IOptions>): Container | undefined {
+    public static async load(tagId: string, params?: RecursivePartial<IOptions>): Promise<Container | undefined> {
         /* elements */
         const domContainer = document.getElementById(tagId);
 
         if (!domContainer) {
             return;
         }
 
+        return this.set(tagId, domContainer, params);
+    }
+
+    /**
+     * Loads the provided options to create a [[Container]] object.
+     * @param id the particles container element id
+     * @param domContainer the dom container
+     * @param params the options object to initialize the [[Container]]
+     */
+    public static async set(id: string, domContainer: HTMLElement, params?: RecursivePartial<IOptions>): Promise<Container | undefined> {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5216054)